src/compiler/transformers/esnext.ts

```diff
@@ -82,6 +82,10 @@ namespace ts {
                     return visitLabeledStatement(node as LabeledStatement);
                 case SyntaxKind.ObjectLiteralExpression:
                     return visitObjectLiteralExpression(node as ObjectLiteralExpression);

```

Comment:
could you add a test case? It would be helpful, since I don't have a good intuition about what kinds of code this will generate yet.